### PR TITLE
#583 Set-DbaTcpPort

### PR DESCRIPTION
**Pull request by HanssonM**
Set-DbaTcpPort, changes the tcpport for specified server

### CODE REVIEW

File: functions/Set-DbaTcpPort.ps1 [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+		[ValidateRange(1, 65535)]
+		[int]$Port,
+		[string[]]$IpAddress
+	)
+	BEGIN
+	{
+		$collection = New-Object System.Collections.ArrayList
+		if ($Port -eq 0 )
+		{
+			throw 'Please specify a specific port using -Port'
```
Comment (wsmelton) [inline functions/Set-DbaTcpPort.ps1]: This is not required since you use `ValidateRange`. If 0 (zero) is provided it will error as violating the range.

Comment (HanssonM) [inline functions/Set-DbaTcpPort.ps1]: Ty, will change that.
Noticed that I made the request to the wrong branch also.

**Comment by potatoqualitee** [comment]
hey @HanssonM - thank you so very much for this PR! My apologies that it's taking so long to merge it. I'll try to get your code to master as soon as possible 👍

**Comment by HanssonM** [comment]
OK @potatoqualitee great!
Nice if I can contribute with something. 😃 
I'm really impressed with everything you've achieved in this project.
Lots of great features that will simplify the daily dbawork.

**Comment by potatoqualitee** [comment]
Thanks again! I'm going to merge, do a patch and Set-DbaTcpPort will be included in the next release! 💃

**Comment by potatoqualitee** [comment]
btw, this makes you a Major Contributor 🍾 ! DM me on Slack/Twitter or email clemaire@gmail.com with your name/pic/website/twitter, and you'll be attributed both in the command and on the help page!! Thanks again for this PR - I saw your account is new, welcome to GitHub, you're kicking butt already :D

**Comment by potatoqualitee** [comment]
hey @HanssonM - You have additional info? You can email or post here. It's for your profile on the team page 👍

**Pull request merged**